client/components/settings-page.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import BottomNavBar from './bottom-nav-bar';
+
+export default class SettingsPage extends React.Component {
+  render() {
+    const username = localStorage.getItem('username');
```

Comment:
Try to pass your user info from App instead. In general, loading things from localstorage isn't good in the `render()` method. It's not horrible in this instance, but we're kind of "going behind react's back" here.